tink/controller/internal/workflow/template_funcs.go

```
@@ -30,3 +33,21 @@ func formatPartition(dev string, partition int) string {
 	}
 	return dev
 }
+
+// netmaskToCIDR converts a netmask (e.g. 255.255.255.0) to CIDR notation (e.g. 24)
+func netmaskToCIDR(netmask string) string {
+	// Parse the netmask
+	ip := net.ParseIP(netmask)
+	if ip == nil {
+		return "24" // Default to /24 if parsing fails
```

The defaulting like this means that errors won't cause a Template to fail to render. This means that invalid subnet masks would most likely have to be troubleshooted when the OS comes up. I think a failure to render the Template, which causes the Workflow to not run, might be an improved user experience. Thoughts?

---

Makes sense. Yeah failing early is definitely preferred